src/test/java/core/basesyntax/ArrayListTest.java

```diff
@@ -7,6 +7,43 @@
 public class ArrayListTest {
     private static final int ELEMENTS_COUNT = 1000;
 
+    // My test
+//    @Test
+//    public void customAddElementToArrayList() {
+//        ArrayList<String> arrayList = new ArrayList<>();
+//        arrayList.add("Test");
+//        arrayList.add(null);
```

Comment:
and we can use /* */ if we want to comment some lines

Reply:
I will simply delete this )